storage/backend/boltoption_darwin.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package backend
+
+import "github.com/coreos/etcd/Godeps/_workspace/src/github.com/boltdb/bolt"
+
+var boltOptions *bolt.Options = nil
```

Comment:
sorry. i should be more specific. I mean boltOpenOptions. `Open` means a lot in the case. Boltdb has other options too.